netbox/templates/generic/object_list.html

```diff
@@ -84,10 +84,14 @@
               <div class="card-body">
                 <div class="float-end">
                   {% if 'bulk_edit' in actions %}
+                    <span class="bulk-edit-button">
                     {% bulk_edit_button model query_params=request.GET %}
+                    </span>
```

Comment:
We should use classes on the buttons directly, rather than wrapping each in a `<span>`.